src/internal/m365/service/groups/export.go

```diff
@@ -33,19 +39,48 @@ func ProduceExportCollections(
 			fp      = restoreColl.FullPath()
 			cat     = fp.Category()
 			folders = []string{cat.String()}
+			coll    export.Collectioner
 		)
 
 		switch cat {
 		case path.ChannelMessagesCategory:
 			folders = append(folders, fp.Folders()...)
-		}
 
-		coll := groups.NewExportCollection(
-			path.Builder{}.Append(folders...).String(),
-			[]data.RestoreCollection{restoreColl},
-			backupVersion)
+			coll = groups.NewExportCollection(
+				path.Builder{}.Append(folders...).String(),
```

Comment:
I know this wasn't your addition, but it suddenly bothers me.
```suggestion
			coll = groups.
			    NewExportCollection(path.Builder{}.Append(folders...).
			    String(),
```

Reply:
This looks like an odd way to split it. Any reason for keeping `String()` in a separate line? Just wanted to make sure you didn't mistype anything.

Reply:
splitting on function chains within builder patterns is pretty standard.  This is effectively:
```go
err := do.
  foo().
  bar().
  baz()
```

Personally, when there's multiple funcs in a chain, I prefer this style over "split on the longest func param.  ie:
```go
// my favorite is above
// second favorite
err := do.
  foo(long, list, of, params).
  bar().
  
// second favorite pt 2
err := do.
  foo(
    even,
    longer,
    param,
    set).
  bar()
  
// not my favorite
err := do.foo(
    params, get, their, own, line,
  ).bar()
```

Reply:
wait a minute, I just realized I read the code wrong.  Yeah, what I suggested is gonzo.  Disregard this suggestion. 🤦